railties/lib/rails/generators/actions.rb

```diff
@@ -216,8 +216,9 @@ def rake(command, options={})
         log :rake, command
         env  = options[:env] || ENV["RAILS_ENV"] || 'development'
         sudo = options[:sudo] && RbConfig::CONFIG['host_os'] !~ /mswin|mingw/ ? 'sudo ' : ''
-        in_root { run("#{sudo}#{extify(:rake)} #{command} RAILS_ENV=#{env}", verbose: false) }
+        in_root { run("#{sudo}#{extify(:rails)} #{command} RAILS_ENV=#{env}", verbose: false) }
```

Comment:
This doesn't sound right to me.. if someone asks us to run `rake`, we should really be running `rake`.